### DISCUSSION TITLE
关于添加站点时配置证书疑惑

### DISCUSSION THREAD

**Discussion by kekylin** [0xJacky/nginx-ui#374] [General]
在添加站点时，不能通过更改证书按钮选取已提前申请好的泛域名证书。需要手动填写证书路径，或者申请新的证书，才能正常配置SSL。请问这是设计如此？还是出现的bug？
![image](https://github.com/0xJacky/nginx-ui/assets/65521882/ce3206ba-fb4c-47aa-a8ca-ffd7fd7d4c75)

**Comment by 0xJacky** [comment]
bug 进展跟踪：https://github.com/0xJacky/nginx-ui/issues/375

**Comment by 0xJacky** [comment]
已修复，请等待下个版本，感谢反馈

**Reply by kekylin** [reply]
谢谢！
